versions-maven-plugin/src/main/java/org/codehaus/mojo/versions/DisplayParentUpdatesMojo.java

```diff
@@ -122,6 +201,56 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         }
     }
 
+    protected ArtifactVersion resolveTargetVersion(String initialVersion)
```

Comment:
I'd move this method to `VersionHelper` (so we would also remove it from `UpdateParentMojo`).